### ISSUE TITLE
Add support for renaming advantage/disadvantage (depends on #59)

### ISSUE THREAD

**Issue by xdy** [xdy/twodsix-foundryvtt#69]
**Is your feature request related to a problem? Please describe.**
Advantage and disadvantage aren't always called that, support the existing system setting to change the names. (i18n is not an appropriate solution, this changes per system, not per language. Though, of course, that setting could well support i18n, but that's a separate issue.)

**Describe the solution you'd like**
Whenever CONFIG.TWODSIX.ROLLTYPES is used, show the appropriate term instead.

**Describe alternatives you've considered**
Just use the current terms and remove the setting. But that seems a shame.

**Additional context**
https://www.drivethrurpg.com/product/207748/Advantage-and-Disadvantage-2e
Issue #59
